src/riscv.h

```diff
@@ -96,6 +96,28 @@ enum {
 #define MSTATUS_MPIE (1 << MSTATUS_MPIE_SHIFT)
 #define MSTATUS_MPP (3 << MSTATUS_MPP_SHIFT)
 
+#define SBI_SUCCESS 0
```

Comment:
Add comments before making list of these SBI related macros. Something like SBI spec.